migrations/20190107123154-create-profile.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Profiles', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    firstName: {
+      type: Sequelize.STRING
+    },
+    lastName: {
+      type: Sequelize.STRING
+    },
+    bio: {
+      type: Sequelize.STRING
```

Comment:
What is the maximum allowed length for the `bio` you are proposing? Are you sure that `STRING` does not have a limit beyond which the user cannot enter more text again?

Check the `Data Types` section of http://docs.sequelizejs.com/manual/tutorial/models-definition.html

And see if `STRING(1234)` or `TEXT` are not preferable

Reply:
Ok. checking it right now